docs/source/theory/kfactors.rst

```diff
@@ -4,8 +4,16 @@ K-Factors
 Another useful tool that `pineko` includes is ``pineko kfactor`` which allows the embedding of a kfactor
 as a proper order in a grid. The usage is the following::
 
-  pineko kfactor GRIDS_FOLDER KFACTOR_FOLDER YAMLDB_PATH TARGET_FOLDER MAX_AS ORDER_EXISTS
+  pineko kfactor GRIDS_FOLDER KFACTOR_FOLDER YAMLDB_FILE TARGET_FOLDER PTO_TO_UPDATE [--order_exists]
 
 where ``GRIDS_FOLDER`` is the folder containing the grids to update, ``KFACTOR_FOLDER`` is the folder
```

Comment:
Consider turning this into a proper list - at the moment it is not really clear

Reply:
Mmm do you mean iterating over datasets?

Reply:
nono, like
```rst
where

- ``GRIDS_FOLDER`` is the folder
- bla
- blub
```

Reply:
done in 5a31fc0674d7fbc004a851f5635af0432bd0ab86